README.md

```diff
@@ -1,30 +1,20 @@
 # Tracker-COVID-19
-Project Description
+This React capstone project is about building a Mobile Web Application to check a list of metrics (numeric values) that I will create making use of React and Redux. Thanks to the COVID-19 metrics I was able to fetch data by using Fetch API, which helped me to get the all the stats per country needed for the project.
 
-![Project Image](./src/assets/images/Tracker-COVID-19.png)
+## Presentation
+https://www.loom.com/share/0b80ccc0813e47be8dd9f0f934e6f381
```

Comment:
- [ ] Good job recording the video presentation, however, I couldn't here anything you said because I guess your microphone has issues or that maybe you didn't turn the microphone on. Kindly record another video and test your microphone properly before recording the video and after kindly play the video to ensure it's audible.

Reply:
Wow, had to change my headphones, sorry guys I really had a lot of trouble, I'm busy so I was just trying to hurry things up. Thanks for patience!